test/uk/gov/hmrc/pensionschemereturn/controllers/PsrOverviewControllerSpec.scala

```diff
@@ -19,35 +19,54 @@ package uk.gov.hmrc.pensionschemereturn.controllers
 import org.mockito.ArgumentMatchers.any
 import org.mockito.MockitoSugar
 import org.scalatest.BeforeAndAfter
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AsyncWordSpec
 import play.api.Application
 import play.api.http.Status
 import play.api.inject.bind
 import play.api.inject.guice.{GuiceApplicationBuilder, GuiceableModule}
 import play.api.libs.json.{JsObject, Json}
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
+import uk.gov.hmrc.auth.core.retrieve.{~, Name}
+import uk.gov.hmrc.auth.core.{AuthConnector, Enrolments}
 import uk.gov.hmrc.http.HeaderCarrier
-import uk.gov.hmrc.pensionschemereturn.base.SpecBase
 import uk.gov.hmrc.pensionschemereturn.services.PsrOverviewService
 import utils.TestValues
 
 import scala.concurrent.Future
 
-class PsrOverviewControllerSpec extends SpecBase with MockitoSugar with BeforeAndAfter with TestValues {
+class PsrOverviewControllerSpec
+    extends AsyncWordSpec
```

Comment:
As a matter of interest, is AsyncWordSpec quicker than PlaySpec  or AnyFreeSpec?

Reply:
In my humble opinion, PlaySpec consists of another traits which we don't need but AsyncWordSpec will be enough for our case here as we need to test Future which is async

Reply:
I was thinking that if it quicker (i.e. executes asynchronously) can we try to use it in the FE repo maybe?